### ISSUE TITLE
HiyaCFW-Helper Error: PLease update 7-zip.

### ISSUE THREAD

**Issue by NateTheTexan** [mondul/HiyaCFW-Helper#67]

**Comment by NateTheTexan** [comment]
I've updated 7-zip to the latest version and it did not help. windows 10, I've tried many different things nothing seems to work. The smaller window says Update 7-zip. Here is the log.
-----------------------------------------------------------
Opening HiyaCFW Helper...
Initializing GUI...
Searching for 7-Zip in the Windows registry...
TWLTool v1.6.1
  by WulfyStylez
  Special thanks to CaitSith2

CTR:
000000: 00 66 02 00 00 9a fd ff ff 99 fd ff 00 00 00 00  .f..............
Normalkey:
000000: 98 ee 80 80 00 6c b4 f6 3a c2 6e 62 f9 ec 34 ad  .....l..:.nb..4.
CTR:
000000: 00 76 02 00 00 8a fd ff ff 89 fd ff 00 00 00 00  .v..............
Normalkey:
000000: 98 ee 80 80 00 6c b4 f6 3a c2 6e 62 f9 ec 34 ad  .....l..:.nb..4.
Nintendo DS rom tool 1.24 - Nov  2 2005 00:24:53 by Rafael Vuijk (aka DarkFader)
TWLTool v1.6.1
  by WulfyStylez
  Special thanks to CaitSith2

reading consoleid/CID from nocashinfo block
Processing twln...
0.00 % complete.
12.14 % complete.
24.28 % complete.
36.42 % complete.
48.56 % complete.
60.70 % complete.
72.84 % complete.
84.98 % complete.
97.11 % complete.
100.00% complete.
Processing twlp...
0.00 % complete.
76.42 % complete.
100.00% complete.
Crypt complete!

ERROR: Data Error : title\00030017\484e4145\content\title.tmd
-------------------------------------------------------------------
